pkg/cable/vxlan/vxlan.go

```diff
@@ -157,7 +157,7 @@ func (v *vxLan) ConnectToEndpoint(endpointInfo *natdiscovery.NATEndpointInfo) (s
 		return "", fmt.Errorf("failed to parse remote IP %s", endpointInfo.UseIP)
 	}
 
-	allowedIPs := parseSubnets(remoteEndpoint.Spec.Subnets)
+	allowedIPs := remoteEndpoint.Spec.ParseSubnets()
```

Comment:
Maybe we should change ParseSubnets to take IPfamily parameter , we can use UseFamily field from  endpointInfo *natdiscovery.NATEndpointInfo